includes/class-sensei-course.php

```diff
@@ -320,6 +320,7 @@ public function course_prerequisite_meta_box_content() {
 			'order'            => 'DESC',
 			'exclude'          => $post->ID,
 			'suppress_filters' => 0,
+			'post_status'      => 'any',
```

Comment:
I don't think we want to include all statuses here, do we? For example, we don't want to show courses that are trashed.

Reply:
Yeap the 'any' option includes everything except the 'trash' and 'auto-draft' stati so this should be ok.